test-apps/ui-test-app/src/frontend/index.tsx

```diff
@@ -394,9 +391,13 @@ export class SampleAppIModelApp {
     let stageId: string;
     const defaultFrontstage = this.allowWrite ? EditFrontstage.stageId : ViewsFrontstage.stageId;
 
-    // Reset QuantityFormatter UnitsProvider with new iModelConnection
-    const schemaLocater = new ECSchemaRpcLocater(iModelConnection);
-    await IModelApp.quantityFormatter.setUnitsProvider (new SchemaUnitProvider(schemaLocater));
+    try {
```

Comment:
@aruniverse  The removed code (without the try/catch) was based on Chris Lawson's latest change to remove the need for the application to have to reset the units provider.  Did you get exceptions and that is why you added the try/catch back in?

Reply:
i think this is left over from a previous merge of mine, ill remove the try/catch